src/components/album/AlbumPreview.js

```diff
@@ -1,9 +1,13 @@
 import React, { useEffect, useState } from 'react';
 import { CardContent } from '@mui/material';
-
+import axiosInstance from '../../apis/axiosClient';
+import WriteReview from '../review/WriteReview';
 const AlbumPreview = ({
+  albumId,
   spotifyUrl,
+  numOfReviews,
```

Comment:
Per previous comment, let's move this to AlbumReview since it's no longer being used here

Reply:
Or maybe we don't need it there anyway if we can check the review api response length instead 😄